libraries/CurieBLE/src/internal/BLECharacteristicImp.h

```diff
@@ -146,18 +146,18 @@ class BLECharacteristicImp: public BLEAttribute{
     /**
      * @brief   Schedule the read request to read the characteristic in peripheral
      *
-     * @param[in]   none
+     * @param[in]   blocked    Flag the call is blocked or un-blocked
      *
      * @return  bool    Indicate the success or error
      *
-     * @note  Only for central device
+     * @note  Only for GATT client
+     *        Default it is block call as per Arduino request
      */
-    bool read();
```

Comment:
So this doesn't change any existing code, right? i.e. if I call `read()` with no arguments, it will block by default, yes?